testsuite/testsuite-buildtool/.gitignore

```diff
@@ -0,0 +1,203 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
I have to remove that. It was copied over from a reproducer contributed by a user.